backend/module/eCampCore/src/EntityService/AbstractEntityService.php

```diff
@@ -181,6 +185,7 @@ public function createQueryBuilder($className, $alias) {
         $this->assertAllowed($entity, __FUNCTION__);
 
         $this->updateEntity($entity, $data);
+        $this->validateEntity($entity);
```

Comment:
Ich habe gerade nicht den Überblick, aber gehe ich richtig in der Annahme dass diese Methode immer in einer Transaktion läuft? Sie ist eben public, ist es also in der Verantwortung des Aufrufers die Transaktion zu starten?

Reply:
Jeder HTTP-Request ist genau eine DB-Transaktion.
Entweder wird alles oder nichts verarbeitet. Es gibt (stand heute) keine "halben" Daten.